### PR TITLE
nimbus.kiln: enable --*-light-client-data

### PR DESCRIPTION
**Pull request by etan-status**

### CODE REVIEW

**Comment by etan-status** [comment]
Will mark this `Ready for review` once the light client feature is supported on `testing` and `stable`.

File: ansible/group_vars/nimbus.kiln.yml [hunk 1]
```diff
@@ -53,6 +53,9 @@ beacon_node_discovery_port: '{{ 9000 + idx|int + 1 }}'
 beacon_node_listening_port: '{{ 9000 + idx|int + 1 }}'
 beacon_node_metrics_port:   '{{ 9200 + idx|int + 1 }}'
 beacon_node_rest_port:      '{{ 9300 + idx|int + 1 }}'
+# Light client server
+beacon_node_serve_light_client_data: true
+beacon_node_import_light_client_data: 'only-new'
```
Comment (jakubgs) [inline ansible/group_vars/nimbus.kiln.yml:58]: Remember to update the variable names here. Also branch name once fix is merged.

**Comment by etan-status** [comment]
- Updated variable names, and marked as `draft` until the kiln branch is up to date and actually has support for the new feature.

**Comment by etan-status** [comment]
- Rename branch to `kiln-dev-auth`.

File: ansible/group_vars/nimbus.kiln.yml [hunk 2]
```diff
@@ -38,8 +38,10 @@ geth_websocket_port: '{{ 9546 + idx|int + 1 }}'
 beacon_node_service_name: 'beacon-node-{{ beacon_node_network }}-devauth-{{ "%02d"|format(idx|int+1) }}'
 beacon_node_network: 'kiln'
 beacon_node_proposer_boosting_debug: true
-# FIXME: Pinned because bf127920 breaks the testnet config.
-beacon_node_repo_branch: 'a35c5f83a8f7880165e19784f73463663db9fdd8'
+beacon_node_repo_branch: 'kiln-dev-auth'
+beacon_node_light_client_data_enabled: true
+beacon_node_light_client_data_serve: true
+beacon_node_light_client_data_import: 'only-new'
```
Comment (jakubgs) [inline ansible/group_vars/nimbus.kiln.yml:44]: Please move the light client variables below the eth1 sync config and separate with a comment for clarity.

Comment (etan-status) [reply ansible/group_vars/nimbus.kiln.yml]: Done

**Comment by etan-status** [comment]
- Move light client variables below eth1 sync config and separate with comment

File: ansible/group_vars/nimbus.kiln.yml [hunk 3]
```diff
@@ -38,12 +38,15 @@ geth_websocket_port: '{{ 9546 + idx|int + 1 }}'
 beacon_node_service_name: 'beacon-node-{{ beacon_node_network }}-devauth-{{ "%02d"|format(idx|int+1) }}'
 beacon_node_network: 'kiln'
 beacon_node_proposer_boosting_debug: true
-# FIXME: Pinned because bf127920 breaks the testnet config.
-beacon_node_repo_branch: 'a35c5f83a8f7880165e19784f73463663db9fdd8'
+beacon_node_repo_branch: 'kiln-dev-auth'
```
Comment (etan-status) [inline ansible/group_vars/nimbus.kiln.yml:41]: `kiln-dev-auth` head is the branch to use for kiln network. The issue highlighted in comment has been fixed:
https://github.com/status-im/nimbus-eth2/blob/kiln-dev-auth/beacon_chain/networking/network_metadata.nim#L208

`kiln-dev-auth` also supports light client data config:
https://github.com/status-im/nimbus-eth2/blob/kiln-dev-auth/beacon_chain/conf.nim#L429-L440

**Comment by jakubgs** [comment]
When I try to deploy this I get:
```
{"lvl":"FAT","ts":"2022-03-28 07:05:59.468+00:00","msg":"Could not load head state, database corrupt?","head":"3f54beeb:1599","tail":"d30e447c:0","genesis":"d30e447c:0"}
```
If I try to start from a fresh DB I get:
```
{"lvl":"FAT","ts":"2022-03-28 07:32:51.131+00:00","msg":"No database and no genesis snapshot found: supply a genesis.ssz with the network configuration, or compile the beacon node with the -d:has_genesis_detection option in order to support monitoring for genesis events","topics":"beacnde"}
```
So the current state of `kiln-dev-auth` is not usable.

**Comment by jakubgs** [comment]
Deployment worked after merging https://github.com/status-im/nimbus-eth2/pull/3549:
```
 > a nimbus.kiln -a "systemctl cat 'beacon-node-*' | grep light-client"      
metal-01.he-eu-hel1.nimbus.kiln | CHANGED | rc=0 >>
    --serve-light-client-data=true \
    --import-light-client-data=only-new \
    --serve-light-client-data=true \
    --import-light-client-data=only-new \
    --serve-light-client-data=true \
    --import-light-client-data=only-new \
    --serve-light-client-data=true \
    --import-light-client-data=only-new \
```

**Review by jakubgs — APPROVED** [review]

**Pull request merged**